exercises/matthew_cumber/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,199 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <iostream>
+#include <iomanip>
+#include <algorithm>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <ctype.h>
+#include "UniqueWord.h"
+
+// Default path for output file reading files from
+std::string const PATH = "/Users/Matt/STFC Placement/C++ Introduction/newstarter/exercises/matthew_cumber/ex01_basics/";
+
+/** Tests if a word has already been found the increments it's count
+ * @param uniqueWords The vector of existing unique words found
+ * @param word The word that is being tested
+ * @return A boolean, true if a word already exists, false otherwise
+ */
+bool inUniqueWords(std::vector<UniqueWord> &uniqueWords,std::string word)
```

Comment:
Since you're not changing the 2nd "word" parameter you can pass this by reference as well for better performed ie const std::string& word